src/backend/db/migrations/1_initial-migrations.sql

```diff
@@ -70,12 +71,44 @@ CREATE TABLE "goal" (
   goal_id UUID PRIMARY KEY DEFAULT uuid_generate_v4(), -- TODO: add index to allow reordering
   iep_id UUID REFERENCES "iep" (iep_id),
   description TEXT NOT NULL,
-  created_at TIMESTAMPTZ NOT NULL DEFAULT NOW()
+  created_at TIMESTAMPTZ NOT NULL DEFAULT NOW(),
+  category TEXT NOT NULL CHECK (category IN ('writing', 'reading', 'math', 'other'))
 );
 
 CREATE TABLE "subgoal" (
   subgoal_id UUID PRIMARY KEY DEFAULT uuid_generate_v4(), -- TODO: add index to allow reordering
   goal_id UUID REFERENCES "goal" (goal_id),
   description TEXT NOT NULL,
-  created_at TIMESTAMPTZ NOT NULL DEFAULT NOW()
+  created_at TIMESTAMPTZ NOT NULL DEFAULT NOW(),
+  instructions TEXT,
+  target_max_attempts INTEGER --How many "questions" to administer in a single sitting
```

Comment:
is a not null value valid here?

Reply:
I think its optional to define a target max, and adding something like `not null default 0` might be confusing/unintuitive? I guess if it was default 0, we just don't render it on the front end. What do you think?

Reply:
if it's optional seems good to me 👍